src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -38,25 +40,49 @@ export const getUniqueValidationSchema = <FormValues,>(
   });
 };
 
-export const getIpAddressValidationSchema = (protocolVersion: 'ipv4' | 'ipv6') => {
-  const protocolVersionLabel = protocolVersion === 'ipv4' ? 'IPv4' : 'IPv6';
+const isValidAddress = (protocolVersion: ProtocolVersion, addressStr: string) => {
+  try {
+    const address =
+      protocolVersion === ProtocolVersion.ipv4
+        ? new Address4(addressStr)
+        : new Address6(addressStr);
+    // ip-address package treats cidr addresses as valid so need to verify it isn't a cidr
+    return !address.parsedSubnet;
+  } catch (e) {
+    return false;
+  }
+};
+
+export const getIpAddressValidationSchema = (protocolVersion: ProtocolVersion) => {
```

Comment:
IIUC, This change will allow entering multiple ip addresses also for fields that are not allowed to have multiple, like default gateway and host ip addresses
I think there should be a specific validation schema for dns ips that enables multiple

Reply:
Done https://github.com/openshift-assisted/assisted-ui-lib/pull/1658/commits/bee1c7f496ba3c730ebe1a6042b71c3334c29254